frontend/src/components/RiderDetail/icons/index.ts

```diff
@@ -0,0 +1,2 @@
+export { default as phone } from './phone.svg';
+export { default as clock } from './clock.svg';
```

Comment:
Centralize the icons for the frontend into one icons folder, removing any duplicates. I believe the same icons might be saved in multiple places.